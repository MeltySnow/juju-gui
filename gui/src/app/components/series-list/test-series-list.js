/* Copyright (C) 2017 Canonical Ltd. */
'use strict';

const React = require('react');
const enzyme = require('enzyme');

const SeriesList = require('./series-list');

describe('SeriesList', function() {

  const renderComponent = (options = {}) => enzyme.shallow(
    <SeriesList
      items={options.items || ['wily', 'kubernetes']} />
  );

  it('can render', function() {
    const wrapper = renderComponent();
    expect(wrapper).toMatchSnapshot();
  });
});
