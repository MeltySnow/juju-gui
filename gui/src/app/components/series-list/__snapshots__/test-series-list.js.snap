// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeriesList can render 1`] = `
<ul
  className="series-list p-inline-list u-no-margin--bottom"
>
  <li
    className="p-inline-list__item series-list__item"
    key="wily"
  >
    wily
  </li>
  <li
    className="p-inline-list__item series-list__item series-list__item--kubernetes"
    key="kubernetes"
  >
    k8s
  </li>
</ul>
`;
